### ISSUE TITLE
Allow customization of SVG fill color

### ISSUE THREAD

**Issue by UtkarshVerma** [dunst-project/dunst#1076]
### Issue description
I use solid SVG icons for my notifications like those from FontAwesome. I wish to have the SVG colored according to the `foreground` colour defined in `dunstrc` as well.

![image](https://user-images.githubusercontent.com/31820255/168771175-6902b916-df97-4f57-84f9-655846092268.png)

Currently, the icon is displayed in black while the text is coloured.

### Installation info

- Version: `Dunst - A customizable and lightweight notification-daemon 1.8.1 (2022-03-02)`
- Install type: `package`
- Window manager / Desktop environment: `dwm`

**Comment by fwsmit** [comment]
Do you know how this could be achieved?

**Comment by UtkarshVerma** [comment]
I don't know how dunst works, but the way I was thinking of implementing this was editing all my SVG icons so that they point to the color `currentColor` instead of a specific value.

Therefore, `<svg fill="currentColor" ...>...</svg>` would do the job. This doesn't work because the SVG render doesn't take this `currentColor` into account. If we could have the SVG renderer define `currentColor` equal to the foreground color, that would be enough.

**Comment by fwsmit** [comment]
Yeah I'm not sure if the svg renderer we use supports setting colours.

**Comment by UtkarshVerma** [comment]
> Yeah I'm not sure if the svg renderer we use supports setting colours.

What if we handle SVGs differently? We can parse SVGs as plain text, do a simple string substitution for "currentColor" -> "foreground", and then pass this modified string to the renderer?

**Comment by HashirGJ8842** [comment]
What is the status of this feature? Can i look into it?

**Comment by fwsmit** [comment]
There was a PR, #1081, but it wasn't finished. Feel free to look into in and perhaps continue the PR. You can also start a new one if you like

**Comment by HashirGJ8842** [comment]
Okay, thanks 👍

**Comment by bynect** [comment]
Note: I liked the implementation of #1081, but I think that it is better to implement a more general solution like `svg_style` for including a stylesheet string into the svg (for more adhoc styles)
